.bingo/Variables.mk

```diff
@@ -53,6 +53,12 @@ $(KIND): $(BINGO_DIR)/kind.mod
 	@echo "(re)installing $(GOBIN)/kind-v0.17.0"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=kind.mod -o=$(GOBIN)/kind-v0.17.0 "sigs.k8s.io/kind"
 
+KUSTOMIZE := $(GOBIN)/kustomize-v5.0.1
```

Comment:
What version of kubectl do we use? This version of kustomize is included in 1.26

Reply:
I don't know if we need to standardize versions, but [operator-controller](https://github.com/openshift/operator-framework-operator-controller/blob/main/.bingo/kustomize.mod) uses v4.5.7 and [catalogd](https://github.com/openshift/operator-framework-catalogd/blob/main/.bingo/kustomize.mod) uses v5.0.1